draft-ietf-mmusic-sdp-bundle-negotiation.xml

```diff
@@ -1261,7 +1261,7 @@ SDP Answer
         which specifies semantics for the usage of a non-zero port value, is
         used. If the stream is marked with port zero in the answer, the answer
         MAY omit all attributes present previously, and MAY list

```

Comment:
Is there a reason why you say "present" here, while elsewhere you say "contain"?